App.jsx

```diff
@@ -1,25 +1,36 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import { useSelector } from 'react-redux'
+import React from "react";
+import { useSelector } from "react-redux";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import HomeScreen from "./components/HomeScreen";
+
+const Stack = createStackNavigator();
 
 const App = () => {
-  const { appTitle } = useSelector(state => state)
+  const { appTitle } = useSelector(state => state);
   return (
-    <View style={styles.container}>
-      <Text>{appTitle}</Text>
-      <StatusBar style="auto" />
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen
+          name="Home"
+          component={HomeScreen}
+          options={{
+            title: appTitle,
+            headerStyle: {
+              backgroundColor: "teal",
+            },
+            headerTitleStyle: {
+              color: "white",
+              fontSize: 30,
+              fontFamily: "AmericanTypewriter-Bold",
+              fontWeight: "bold",
+              textAlign: "center"
+            },
+          }}
+        />
+      </Stack.Navigator>
+    </NavigationContainer>
   );
-}
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'left',
-    justifyContent: 'center',
-  },
-});
+};
 
-export default App
+export default App;
```

Comment:
Why did you removed the stylesheet from App.jsx?

Reply:
We moved it to the Homescreen component so to have less code in App